src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs

```diff
@@ -18,6 +18,12 @@ public class UserStoppedTypingBehavior : BaseBehavior<InputView>
 		public static readonly BindableProperty CommandProperty
 			= BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(UserStoppedTypingBehavior));
 
+		/// <summary>
+		/// Backing BindableProperty for the <see cref="CommandParameter"/> property.
+		/// </summary>
```

Comment:
😍

Reply:
@jfversluis will be in the next nightly?

Reply:
It's not merged yet so no